optimize/backend/src/main/java/io/camunda/optimize/service/db/es/report/interpreter/util/AggregateByDateUnitMapperES.java

```diff
@@ -1,77 +0,0 @@
-/*
- * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
- * one or more contributor license agreements. See the NOTICE file distributed
- * with this work for additional information regarding copyright ownership.
- * Licensed under the Camunda License 1.0. You may not use this file
- * except in compliance with the Camunda License 1.0.
- */
-package io.camunda.optimize.service.db.es.report.interpreter.util;
-
-import co.elastic.clients.elasticsearch._types.aggregations.CalendarInterval;
-import io.camunda.optimize.dto.optimize.query.report.single.group.AggregateByDateUnit;
-import java.time.temporal.ChronoUnit;
-
-public final class AggregateByDateUnitMapperES {
```

Comment:
unrelated but decided to delete this class as it was only used in one place, rather than being a real utility